components/jupyter-web-app/kubeflow_jupyter/default/templates/notebooks.html

```diff
@@ -13,7 +13,7 @@
     <select class="mdl-textfield__input" id="ns-select" name="ns">
       <!-- Fill with the namespaces the user can see -->
       {% for ns in namespaces %}
-        <option value="{{ ns }}">{{ ns }}</option>
+        <option value="{{ ns }}" {{ ns == 'kubeflow'|yesno:"selected" }}>{{ ns }}</option>
```

Comment:
This cherry-pick is from a PR that had a syntax error.


You can use:
```suggestion
        <option value="{{ ns }}" {% if ns == 'kubeflow' %}selected{% endif %}>{{ ns }}</option>
```

Reply:
👍